client/src/services/firebase.ts

```diff
@@ -0,0 +1,20 @@
+import firebase from "firebase/app";
+//limit import to auth (keep comment to keep linter happy)
+import "firebase/auth";
+
+const firebaseConfig = {
+  apiKey: "AIzaSyApDXStJREUsZymQBGs4yJXi22wc3PBvSo",
+  authDomain: "bp-pregnancy-centre.firebaseapp.com",
+  projectId: "bp-pregnancy-centre",
+  storageBucket: "bp-pregnancy-centre.appspot.com",
+  messagingSenderId: "907701825043",
+  appId: "1:907701825043:web:e967c13a3543cbd8a219e6",
+};
+try {
+  firebase.initializeApp(firebaseConfig);
+} catch (err) {
+  if (!/already exists/.test(err.message)) {
```

Comment:
Is there an error provided by the firebase sdk? If the error message string changes, then we would have to debug and rewrite this condiitonal 

https://github.com/firebase/firebase-js-sdk/blob/master/packages/auth/src/error_auth.js#L123

Reply:
Switched to err.code!